spring-boot-admin-server/src/main/java/de/codecentric/boot/admin/server/config/AdminServerNotifierAutoConfiguration.java

```diff
@@ -61,6 +65,22 @@
 @AutoConfigureAfter({MailSenderAutoConfiguration.class})
 public class AdminServerNotifierAutoConfiguration {
 
+    @Configuration(proxyBeanMethods = false)
+    @ConditionalOnMissingBean(name = "notifierRestTemplate")
+    @ConditionalOnBean(Notifier.class)
+    public static class NotifierRestTemplateConfiguration {
+        @Bean
+        @ConfigurationProperties("spring.boot.admin.notify.proxy")
+        public NotifierProxyCustomizer notifierProxyCustomizer() {
+            return new NotifierProxyCustomizer();
+        }
+
+        @Bean
+        public RestTemplate notifierRestTemplate() {
```

Comment:
I don't think it's not a good idea to add a RestTemplate bean as it may conflict with the users' configuration.
They may have afterwards two beans of the exact type and are forced to use `@Qualifiers`. Imho I thinks it's ok to do the instantiation of the RestTemplate in each notifiers configuration...

Reply:
Ok, got your argument...  Better now?

What do you think about proxy authentication? Should we support it out of the box? Would need a new dependency 
`<dependency>
    <groupId>org.apache.httpcomponents</groupId>
    <artifactId>httpclient</artifactId>
</dependency>
 `

Reply:
Hmm... I'd wait for that until we have a demand. "Kratzen wenn es juckt"